dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkData.ts

```diff
@@ -200,6 +198,24 @@ describeIf(
       cy.ensureLoggedIn("data_uploader", Cypress.env("KEYCLOAK_UPLOADER_PASSWORD"));
     });
 
+    it(
+      "Upload a company without uploading framework data for it and check if it neither appears in the + " +
+        "autocomplete suggestions nor in the search results, even though no framework filter is set.",
+      () => {
+        const companyName = "ThisCompanyShouldNeverBeFound12349876";
```

Comment:
Solche sachen wollten wir ja eigentlich aus den FakeFixtures holen oder nicht?

Reply:
Hier muss eine Company erstellt werden, deren CompanyName - auch als substring - keiner Company zugehörig ist, für die bereits FrameworkData hochgeladen wurde.
Da durch die Prepopulation jedoch für alle companies auch direkt ein framework dataset hochgeladen wird, kann ich an dieser Stelle keinen companyName verwenden, der ein substring der company names in den fakefixtures ist.

Reply:
Ok